test/inductor/test_fx_fusion.py

```diff
@@ -66,7 +67,7 @@ def test_kwarg3(x, y):
         ]
         for f in [test_kwarg, test_arg, test_arg2, test_kwarg2, test_kwarg3]:
             traced = trace_func(f, inputs)
-            self.assertTrue(torch.allclose(f(*inputs), traced(*inputs)))
+            self.assertTrue(same(f(*inputs), traced(*inputs)))
```

Comment:
Why not do something like that? And if lower tolerance is specific to arm platform, why not condition it on `platform.processor() == 'aarch64'`?
```suggestion
            self.assertEqual(f(*inputs), traced(*inputs), rtol=1e-4, atol=1e-4)
```

Reply:
I have fixed the lint and improved this by using `[torch.testing.assert_close](https://pytorch.org/docs/stable/testing.html#torch.testing.assert_close)` which seems to be the recommended approach. The default tolerances appears to satisfy all hardware types so no need to put a platform condition in.